change/@microsoft-teams-js-d43a230c-d15c-486e-82c4-c72eb6ad2e61.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding isSupported Check to logs capability",
```

Comment:
nit: please change "Check" to lowercase "check" for consistency across change files

Reply:
Made changes in the latest commit.